modules/mod-deep-learning/ModelCardPanel.cpp

```diff
@@ -0,0 +1,517 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+
+   ModelCardPanel.cpp
+   Hugo Flores Garcia
+
+******************************************************************/
+
+#include "DeepLearningEffectBase.h"
```

Comment:
A good rule to follow is that each .cpp file will # include its corresponding h file before any other, so that we are sure each .h file is compiled at least once in the system as the first header, and therefore contains all other nested inclusions that it depends on.  This forces the # include directives to tell the complete story of compilation dependencies, and you know I care about getting a correct picture of those dependencies.

You followed that rule but with two exceptions.  This is one, and ModelManager.cpp is the other.

I rearranged the # include lines in my own build, to be sure things still compile.  They do.

Please make those changes too.

Reply:
resolved in 548b8a32d77953433c6d4025cc08a10c7af4d98e